fe1-web/src/features/digital-cash/screens/PoPTokenScanner.tsx

```diff
@@ -70,10 +71,10 @@ const PoPTokenScanner = () => {
     });
   };
 
-  const onScanData = (popToken: string | null) => {
-    if (popToken) {
-      if (tokenMatcher.test(popToken)) {
-        goBack(popToken);
+  const onScanData = ({ data }: BarCodeScanningResult) => {
```

Comment:
And this would allow us to revert this change too. 
No reason for `PoPTokenScanner` to know the details of how the scanner is implemented.